code/game/objects/effects/meteors.dm

```diff
@@ -253,7 +253,8 @@ GLOBAL_LIST_INIT(meteors_ops, list(/obj/effect/meteor/goreops)) //Meaty Ops
 	..()
 	explosion(loc, 0, 0, 4, 3, 0)
 	new /obj/effect/decal/cleanable/greenglow(get_turf(src))
-	radiation_pulse(src, 500)
+	radiation_pulse(src, 5000, 7)
```

Comment:
make sure to test this!

Reply:
This was tested. I'll give some pictures of the impacts with it shortly:

It's... generally underwhelming, to be honest.